omnigibson/prims/rigid_prim.py

```diff
@@ -101,7 +101,7 @@ def _post_load(self):
         self.update_meshes()
 
         # Possibly set the mass / density
-        if len(self._collision_meshes) == 0:
+        if not self.has_collision_meshes:
             # We have no collision meshes, this is a meta (virtual) link, so set a negligible mass
             self.mass = 1e-6
```

Comment:
maybe we should also explicitly set density = 0 here?

Reply:
done